devops/templates/create-env-step-template.yml

```diff
@@ -14,6 +14,10 @@ parameters:
   default: interpret_conda_env
 
 steps:
+  - bash: conda update conda
+    displayName: Update conda environment due to known bug on windows
+    condition:  eq(variables['Agent.OS'], 'Windows_NT')
```

Comment:
Is the condition really needed? Surely upgrading Conda shouldn't do any _harm_ on the other platforms?

Reply:
Yes, I wanted to only update windows since issues are limited to that - once the default one on windows no longer has this issue I will remove this upgrade.  Don't want to update the default linux/mac since they seem to be working and who knows what getting the latest might break.